app/main/db/queries.py

```diff
@@ -7,28 +7,32 @@
 from app.main.util.date_formatter import validate_date_range
 
 
-def build_fuzzy_search_query(
-    query_string: str, transferring_body_id=None, sorting_orders=None
-):
-    filter_value = str(f"%{query_string}%").lower()
-
-    fuzzy_filters = or_(
-        func.lower(Consignment.ConsignmentReference).like(filter_value),
-        func.lower(Body.Name).like(filter_value),
-        func.lower(Body.Description).like(filter_value),
-        func.lower(Series.Name).like(filter_value),
-        func.lower(Series.Description).like(filter_value),
-        func.lower(File.FileName).like(filter_value),
-    )
+def _get_search_terms(query_string):
+    search_terms = []
+    query_terms = query_string.strip().split(",")
+    # exclude any empty items
+    query_terms = list(filter(None, query_terms))
+
+    for i in range(len(query_terms)):
+        if len(query_terms[i].strip()) > 0:
+            search_terms.append("%" + query_terms[i].strip().lower() + "%")
+    return search_terms
 
+
+def build_fuzzy_search_transferring_body_query(
+    query_string: str, transferring_body_id, sorting_orders=None
+):
     sub_query = (
         db.session.query(
             Body.BodyId.label("transferring_body_id"),
             Body.Name.label("transferring_body"),
+            Body.Description.label("transferring_body_description"),
             Series.SeriesId.label("series_id"),
             Series.Name.label("series"),
+            Series.Description.label("series_description"),
             Consignment.ConsignmentId.label("consignment_id"),
             Consignment.ConsignmentReference.label("consignment_reference"),
+            File.FileId.label("file_id"),
```

Comment:
why was file_id added?

Reply:
File_Id needed for the record page link as needs to pass in FileId as parameter, at some point during the query change we would have removed the column, 

![image](https://github.com/nationalarchives/da-ayr-beta-webapp/assets/107114665/5ad28ad0-78bd-4499-b76d-dc3ca7acecb5)

Reply:
ah right, cheers